src/api.js

```diff
@@ -174,6 +174,13 @@ export const MOCK_EVENT = {
 	yes_rsvp_count: 23
 };
 
+export const MOCK_BEST_TOPIC = {
```

Comment:
From what I can tell, a topic returned from the 'best_topics' option is distinct (has fewer props) from a topic returned from the top-level topic endpoint.

However, I did rename this.